app/views/cooks/index.html.erb

```diff
@@ -23,6 +23,9 @@
       <td><%=cook.name%></td>
       <td><%=cook.cuisine%></td>
       <td><%=cook.city%></td>
+      <td><%= link_to 'Show', cook %></td>
```

Comment:
Would this link be better around the name?

Reply:
Thank you very much, I will look into this.